product-scenarios/4-single-sign-on/4.1-sso-for-web-app/4.1.4-sso-with-oidc/4.1.4.3-basic-profile/src/test/resources/dcr-requests/dcr-request-2.json

```diff
@@ -4,6 +4,6 @@
     "authorization_code"
   ],
   "redirect_uris": [
-    "https://sso-app-oidc-code-grant-2.com/callback"
+    "http://localhost"
```

Comment:
don't we need port here? if we don't give the port, to which port it is returning

Reply:
It just needs a valid host to obtain the callback.